cmd/gpu_levelzero/zes.c

```diff
@@ -138,7 +138,10 @@ static ze_result_t enumerate_zes_devices(void)
         zes_device_handle_t dev_h = zes_handles[i];
 
         zes_pci_properties_t pci_props;
-        if (zesDevicePciGetProperties(dev_h, &pci_props) != ZE_RESULT_SUCCESS) {
+        memset(&pci_props, 0, sizeof(pci_props));
```

Comment:
With C23: https://en.cppreference.com/w/c/language/struct_initialization.html

You could do just
```suggestion
        zes_pci_properties_t pci_props = {};
```
But AFAIK this, supported by C99, is enough:
```suggestion
        zes_pci_properties_t pci_props = {.pNext = NULL};
```